src/CallInvite.tsx

```diff
@@ -370,21 +371,41 @@ export class CallInvite extends EventEmitter {
    *    - Rejects when the message is unable to be sent.
    */
   async sendMessage(message: CallMessage): Promise<OutgoingCallMessage> {
+    if (!(message instanceof CallMessage)) {
+      throw new InvalidArgumentError(
+        'Argument "message" must be instanceof "CallMessage"'
+      );
+    }
+
+    if (
+      typeof message.getContent() === 'undefined' ||
+      message.getContent() === null
+    ) {
+      throw new InvalidArgumentError('"content" is empty');
+    }
+
+    //@ts-ignore
+    if (message.getMessageType().length === 0) {
+      throw new InvalidArgumentError(
+        '"messageType" must be a non-empty string'
+      );
+    }
+
```

Comment:
What do you think about moving this validation logic to a reusable function? Even a method on the `CallMessage` object perhaps?